codegenerator/cli/templates/static/codegen/src/Benchmark.res

```diff
@@ -187,8 +255,7 @@ module Summary = {
     @as("std-dev") stdDev: float,
     min: float,
     max: float,
-    last: float,
```

Comment:
I actually never meant to keep last in before. I don't think it's very useful. And it won't work with this model of accumulated data rather than saving all data points in the data set.